### DISCUSSION TITLE
Can I open programs with programs that I added to phoenicis?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2350] [Q&A]
I wanna open a program with a wine program that I added to phoenicis.

**Comment by plata** [comment]
I don't understand the question. Can you give an example?

**Reply by ghost** [reply]
So if you have a .xdelta file for example.
There is no gui program to open it in linux afaik,but there is one for windows,called "xdelta ui"
In windows you would right click the .xdelta file and select "open with xdelta ui",I wanna do the exact same with that program added to phoenicis.

**Reply by plata** [reply]
https://github.com/PhoenicisOrg/phoenicis/issues/258

**Reply by ghost** [reply]
I don't understand that really.
What the hack are mime links and does that do the thing I want?

**Reply by plata** [reply]
It basically says "How shall I open files of type image?" (for all kinds of type).

**Reply by ghost** [reply]
So if I got it right then it's the same as doing "open with..." in a file manager?

**Reply by plata** [reply]
Somewhat. Except that it picks the correct program automatically. E.g. you double click a png and it will open Gwenview.
